config/canonical.json

```
@@ -31325,7 +31328,9 @@
     "tags": {
       "brand": "Supermercado Dia",
       "name": "Supermercado Dia",
-      "shop": "supermarket"
+      "shop": "supermarket",
+      "brand:wikidata": "Q925132",                     
+      "brand:wikipedia": "en:Dia_(supermarket_chain)" 
```

Can you remove the `_` underscores?  thanks!
```suggestion
      "brand:wikipedia": "en:Dia (supermarket chain)" 
```